internal/app/evaluator/defaulteval/evaluator.go

```diff
@@ -30,16 +35,35 @@ var (
 		"app":       "evaluator",
 		"component": "evaluator.default",
 	})
+
+	collidedMatchesPerEvaluate     = stats.Int64("openmatch.dev/evaluator/collided_matches_per_call", "Number of collided matches per default evaluator call", stats.UnitDimensionless)
+	collidedMatchesPerEvaluateView = &view.View{
+		Measure:     collidedMatchesPerEvaluate,
+		Name:        "openmatch.dev/evaluator/collided_matches_per_call",
+		Description: "Number of collided matches per default evaluator call",
+		Aggregation: view.Sum(),
+	}
 )
 
 type matchInp struct {
 	match *pb.Match
 	inp   *pb.DefaultEvaluationCriteria
 }
 
+// BinderServices define the initialization steps for this evaluator
+func BinderServices() appmain.Bind {
```

Comment:
Like the other apps, no need for a lambda, just declare the function as:
```
func BindService(p *appmain.Params, b *appmain.Bindings) error {
```
Then reference with `evaluator.BindService`, instead of `evaluator.BindServices()`